### ISSUE TITLE
Feature request: interactive ruckus in drracket

### ISSUE THREAD

**Issue by yminsky** [cbiffle/ruckus#1]
I'm super excited to see Ruckus, and would love to use it in teaching my kids. I'm wondering if there's any hope of getting Ruckus to work in an interactive mode similar to what OpenSCAD does, where when you hit #R, you immediately get a visualization of your model to pop up.

**Comment by cbiffle** [comment]
It's certainly possible!  I just haven't put the work in.

My workflow in Ruckus is to edit in `vi` and hit F5 to reload the design.  (This is also my workflow in OpenSCAD; I don't care for the internal text editor in either OpenSCAD or DrRacket.)

**Comment by yminsky** [comment]
Yeah, I mostly want it for using with my kids, for whom the DrRacket IDE is a real boon.

**Comment by cbiffle** [comment]
That makes sense.  (I know *adults* who love DrRacket -- no judgment.  It's just not something I've put work into.)

I'm going to leave this open because it would be a great addition, but I'm not personally likely to get time to hack on it soon.
